proxy/src/telemetry/metrics/labels.rs

```diff
@@ -88,6 +125,153 @@ pub struct DstLabels {
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub struct TlsStatus(ctx::transport::TlsStatus);
 
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub enum TlsConfigLabels {
+    Reloaded,
+    InvalidTrustAnchors,
+    InvalidPrivateKey,
+    InvalidEndEntityCert,
+    Io { path: PathBuf, errno: Option<Errno>, },
+}
+
+mk_err_enum! {
+    /// Taken from `errno.h`.
+    #[allow(non_camel_case_types)]
+    #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+    enum Errno from i32 {
+        1 => EPERM,
```

Comment:
Why not use `libc::EPERM`, et al., instead? Then we don't have to worry if the numbers are correct.